vite.config.js

```diff
@@ -37,13 +37,13 @@ export default defineConfig({
     sourcemap: 'inline',
   },
   test: {
-    environment: 'jsdom',
+    environment: 'js',
     globals: true,
-    setupFiles: 'src/setupTests.ts',
+    setupFiles: 'src/setupTest.ts',
```

Comment:
**Suggestion:** Correct the environment to 'jsdom' and setupFiles to 'src/setupTests.ts'.
```suggestion
    environment: 'jsdom',
    setupFiles: 'src/setupTests.ts',
```